website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,167 @@
+## Usage
+
+### When to use
+
+- When the list of options is large and search is required to help filter the list.
+- When multiple selections are made and displayed as a group.
+- When you require complex structured content to be displayed within the list (and trigger).
+
+### When not to use
+
+- To display a list of links (or buttons) in a menu, consider the [HDS Dropdown](/components/dropdown) component instead.
+- A list of options 5 or less, consider the [HDS Select](/components/form/select) form element.
+- As a means to filter an array of objects within a data set; e.g., when presented in a tabular format or list. Instead, consider other filtering patterns.
+
+
+## Variants
+
+### Single
+Selection is limited to a single list item.
+
+![Single selection example with list open](/assets/components/form/super-select/super-select-single-variant-text.png =504x*)
+
+### Multiple
+Multiple selections can be made within the list.
+
+![Multiple selection example with list open](/assets/components/form/super-select/super-select-multiple-variant.png =504x*)
+
+## Trigger default states
+
+**Empty**
+The default option, where the trigger is empty, showing nothing selected.
+
+![Showing an empty trigger, indicating to the user there is no option selected yet](/assets/components/form/super-select/super-select-trigger-empty.png =399x*)
+
+**Placeholder text**
+Placeholder text was added to the trigger, which helps the user understand additional context, but is not reliant on this information to carry out a task.
+
+![Showing placeholder text inside of the trigger, still indicating no selection is made, but with some additional guidance](/assets/components/form/super-select/super-select-trigger-placeholder-text.png =399x*)
+
+**Populated**
+The trigger shows information populated, and has an option pre-selected before user interaction.
+
+![Showing preselected options in both a single and multiple variant. This helps show something has been preselected previously and the user can update their options](/assets/components/form/super-select/super-select-trigger-filled.png =399x*)
+
+## Before and after options
+The before and after options are areas that sit above and below the list respectively, independent from the list’s overflow. They allow for either predefined components (like search) or custom content to occupy these spaces.
+
+### BeforeOptions
+BeforeOptions allows for one of two options to be displayed at a given time: search or custom content. These options are the same between both single and multiple variants.
```

Comment:
```suggestion
BeforeOptions allows for either a search field or custom content to be displayed before the options list.
```

Reply:
Since "Single" and "Multiple" here refer explicitly to the HDS components I think both should be capitalized.

(@jorytindall @heatherlarsen @andrew-hashicorp could any of you confirm that this is correct?)